java/ql/src/semmle/code/java/frameworks/jackson/JacksonSerializability.qll

```diff
@@ -63,6 +67,23 @@ private class JacksonReadValueMethod extends Method, TaintPreservingCallable {
   override predicate returnsTaintFrom(int arg) { arg = 0 }
 }
 
+/**
+ * A method used for creating a `JsonParser` object using Jackson. The first parameter is the object to
+ * create the `JsonParser` from.
+ */
+private class JacksonCreateParserMethod extends Method, TaintPreservingCallable {
+  JacksonCreateParserMethod() {
+    (
+      getDeclaringType().hasQualifiedName("com.fasterxml.jackson.core", "JsonFactory") or
+      getDeclaringType().hasQualifiedName("com.fasterxml.jackson.databind", "ObjectMapper") or
+      getDeclaringType().hasQualifiedName("com.fasterxml.jackson.databind", "ObjectReader")
```

Comment:
This class isn't referenced anywhere, so might as well use csv-rows and add these three steps to the `private class JacksonModel extends SummaryModelCsv` below.

Reply:
Good catch, thanks! Done in d9fb650